Makefile.dep

```diff
@@ -130,3 +130,9 @@ ifneq (,$(filter nhdp,$(USEMODULE)))
 	USEMODULE += oonf_common
 	USEMODULE += oonf_rfc5444
 endif
+
+ifneq (,$(filter stl-compatibility,$(USEMODULE)))
```

Comment:
Better call it cpp11-compat or anything similar. → http://stackoverflow.com/q/5205491/416224